wiki/Legal/Copyright/Music_Licensing/en.md

```diff
@@ -0,0 +1,23 @@
+---
+legal: true
+---
+
+# Music Licensing
+
+Artists who enter a licensing agreement with osu! are listed under the [Featured Artists](/wiki/Featured_Artists) program. The osu! team regularly reaches out to artists that the community loves and new artists whose music is appropriate for use within the community and game.
+
+Artists interested in joining the Featured Artist program are encouraged to contact osu! for licensing negotiations.
+
+## Terms
+
+Music licensed by osu! is freely available for download from the [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) in `.osz` files, which are level (beatmap) templates. A one-minute automatically generated preview of each track is available for streaming on its artist's page. Because songs are used as a catalog for community members to create beatmaps, some of them may not be immediately used as in-game content.
+
+We look to cover usage within the game and related website, which does not include further usage on other streaming platforms (e.g. user uploads to YouTube). One exception is that we utilise Featured Artist tracks as background music during official events, such as [osu! World Cup](/wiki/Tournaments#-osu!-world-cup) live-streams. An exclusion clause can be negoatiated on request to completely restrict usage to within the game only.
```

Comment:
```suggestion
We look to cover usage within the game and related website, which does not include further usage on other streaming platforms (e.g. user uploads to YouTube). One exception is that we utilise Featured Artist tracks as background music during official events, such as [osu! World Cup](/wiki/Tournaments#-osu!-world-cup) live-streams. An exclusion clause can be negotiated on request to completely restrict usage to within the game only.
```

Reply:
I thought this was fixed cuz I saw "negotiated" written correctly somewhere else in the article
rip